.changes/state_control.md

```diff
@@ -0,0 +1,5 @@
+---
+"window-vibrancy": "minor"
+---
+
+Expose vibrancy state control for macos, default is follow window active state.
```

Comment:
Add a 3rd argument to `apply_vibrancy` to specify the vibrancy state, follows the window active state by default.